curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -900,8 +900,6 @@ private void sendBackgroundResponse(int rc, String path, Object ctx, String name
 
     private static CuratorEvent makeCuratorEvent(CuratorFrameworkImpl client, int rc, String path, Object ctx, String name, Stat stat)
     {
```

Comment:
Let's inline this method since it's now only a delegate.

Reply:
I have introduced a static version of `sendBackgroundResponse`, so that all response are go through this method finally. I saw no "beauty" code path to construct `CuratorEvent`, too many parameters ...